src/components/layout/Page.tsx

```diff
@@ -25,19 +25,19 @@ const StyledPage = styled(Container)`
 const PageMeta = () => {
   const { pathname } = useLocation()
   const cakePriceUsd = usePriceCakeBusd()
-  const cakePriceUsdDisplay =
-    cakePriceUsd.gt(0) &&
-    `$${cakePriceUsd.toNumber().toLocaleString(undefined, {
-      minimumFractionDigits: 3,
-      maximumFractionDigits: 3,
-    })}`
+  const cakePriceUsdDisplay = cakePriceUsd.eq(0)
+    ? ''
+    : `$${cakePriceUsd.toNumber().toLocaleString(undefined, {
+        minimumFractionDigits: 3,
+        maximumFractionDigits: 3,
+      })}`
   const pageMeta = customMeta[pathname] || {}
   const { title, description, image } = { ...DEFAULT_META, ...pageMeta }
-  const titleWithPrice = [title, cakePriceUsdDisplay].join(' - ')
+  const pageTitle = cakePriceUsdDisplay ? [title, cakePriceUsdDisplay].join(' - ') : title
```

Comment:
Was going to comment that we could just keep the original `cakePriceUsdDisplay` (with `gt(0) && ...`) and change only this line (36). But then figured out that with this change `cakePriceUsdDisplay` would be nice `string` type instead of `bool | string` 👍